docs/development_environment.md

```diff
@@ -82,6 +82,14 @@ To isolate your environment from the rest of the system, set up a [`venv`](https
 $ python3 -m venv .
 $ source bin/activate
 ```
+
+Update `pip` itself
+
+```bash
+$ pip install --upgrade pip
```

Comment:
This is not necessary. If pip is outdated, it will be updated as part of the Home Assistant installation (it is part of our requirements)

Reply:
In my case, I met some error during the script/setup indicated my pip is outdated. I manually upgraded pip to resolve them.

Reply:
@balloob I see what you mean. There is a pip requirement in setup.py `pip>=8.0.3`, it may need to adjust a little bit. I will withdraw this part of change from document, and try to figure out what package need higher version of pip in my case.

But I want to keep python36 change, since aiphttp3 simply need python 3.5.3+